packages/widgets/src/components/PieChart/PieChart.stories.tsx

```diff
@@ -0,0 +1,143 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { css, useTheme } from '@emotion/react';
+import { MainColors, Typography } from '@ssa-ui-kit/core';
+
+import { PieChart, PieChartLegend } from './index';
+import { fitnessData, accountData } from './stories/fixtures';
+
+export default {
+  title: 'Widgets/PieChart',
+  component: PieChart,
+} as Meta<typeof PieChart>;
+
+export const FitnessExample: StoryObj<typeof PieChart> = () => {
+  const theme = useTheme();
+  const colorNames = ['blueLight', 'turquoise'] as unknown as Array<
+    keyof MainColors
+  >;
+  const pieChartColors = [
+    theme.colors.blueLighter,
+    theme.colors.turquoise,
+  ] as unknown as string[];
+
+  return (
+    <div style={{ width: '400px' }}>
+      <PieChart
+        data={fitnessData}
+        colors={pieChartColors}
+        animate={false}
+        title={
+          <Typography
+            variant="body2"
+            weight="regular"
+            color={theme.colors.greyDarker60}
+            css={css`
+              font-size: 16px;
+              line-height: 16px;
+              margin-top: -5px;
+            `}>
+            Total
+            <Typography
+              variant="body2"
+              weight="bold"
+              color={theme.colors.greyDarker}
+              css={css`
+                font-size: 27.65px;
+                line-height: 35px;
+              `}>
+              143
+              <Typography
+                variant="body2"
+                weight="regular"
+                as="span"
+                color={theme.colors.greyDarker60}
+                css={css`
+                  font-size: 16px;
+                  font-weight: 600;
+                  margin-left: 3px;
+                `}>
+                hrs
+              </Typography>
+            </Typography>
+          </Typography>
+        }>
+        <PieChartLegend data={fitnessData} colors={colorNames} />
+      </PieChart>
+    </div>
+  );
+};
+FitnessExample.args = {};
+
+export const AccountExample: StoryObj<typeof PieChart> = () => {
+  const theme = useTheme();
+  const colorNames = [
+    'yellow',
+    'blue',
+    'green',
+    'yellowWarm',
+    'blueLight',
+    'turquoise',
+  ] as unknown as Array<keyof MainColors>;
```

Comment:
Could we use only the one place for colors list?

Reply:
done